draft-yoshino-wish-00.txt

```diff
@@ -67,12 +67,14 @@ Table of Contents
    2.  Background  . . . . . . . . . . . . . . . . . . . . . . . . .   3
    3.  Conformance Requirements and Terminology  . . . . . . . . . .   4
    4.  WiSH Protocol . . . . . . . . . . . . . . . . . . . . . . . .   4
-   5.  Framing . . . . . . . . . . . . . . . . . . . . . . . . . . .   4
-   6.  Acknowledgements  . . . . . . . . . . . . . . . . . . . . . .   5
-   7.  References  . . . . . . . . . . . . . . . . . . . . . . . . .   5
-     7.1.  Normative References  . . . . . . . . . . . . . . . . . .   5
-     7.2.  Non-normative References  . . . . . . . . . . . . . . . .   6
-   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .   6
+   5.  Subprotocol Negotiation . . . . . . . . . . . . . . . . . . .   4
```

Comment:
I suggest we have two appendix-type sections, after the framing section.

5. Framing
6. Use WiSH over HTTP
7. WebSocket compatiblity (how to bridge WiSH and WebSocket)